core/controlplane/config/templates/cloud-config-controller

```diff
@@ -791,6 +791,116 @@ write_files:
       sed -i -e "s#\$ETCDKEY#$etcd_key#g" /srv/kubernetes/manifests/calico.yaml
 
 {{ end }}
+{{ if .ExportKubeResources }}
+  - path: /srv/kubernetes/manifests/cluster-dump.yaml
```

Comment:
And I'd also expect the `cluster-dump`s to be changed to `kube-resources-autosave` accordingly.